doc/gov/chapters/chapter02.md

```diff
@@ -13,26 +13,36 @@
 
 
 
 <a name="2.2"></a>
 ## 2.2 Request Onboarding Info Packet
 
-[Welcome Message Here]
+[Sample Welcome Message Here] 
+Welcome to the CNTT group –
+
+It is wonderful that you, [Company and person] have decided to join in the effort to make building the infrastructure to support Telecom workloads more robust and standardized. We believe that these efforts will benefit not only the operators, but also the vendors, and ultimately all our customers as well, by hopefully making it easier and simpler to incorporate new technology into our systems.
+
+To get you started, I encourage you to go to our Getting Started page to download a packet of information on how to get engaged at the working level with CNTT.  We also have materials in the Wiki and Github repositories, so I hope that all this doesn’t overwhelm you.  We are excited that you will be able to become more engaged in our efforts and as always, we will be happy to help you get up to speed.  
+
+In addition to the many on-line work streams and workstream related on-line meetings, we also have periodic face to face workshops, which we find are invaluable for speeding the delivery of CNTT’s desired outcomes. More details on upcoming events can be found on the CNTT wiki. 
+
+Feel free to reach out at any time, I am sure we will be more than happy to answer any questions you might have. Welcome again and looking forward to working with you.
+
 
 - **Guide To Getting Started**
 
   - **How do I start the process of becoming a participant?** Onboarding starts as soon as you submit a request to join CNTT via email to cntt-gov@lists.opnfv.org. You will receive additional information such as links to the CNTT Getting Started wiki and additional onboarding materials.
   
-  - **How long will it take to onboard?** The onboarding process will take approximately one hour of your time initially.
+  - **How long will it take to onboard?** The onboarding process will initially take approximately one hour.
   
-  - **What will I learn about during onboarding process?** You’ll get an overview of the CNTT mission and goals, community organizational structures and processes, as well as technology development status and roadmaps. Additionally, you’ll be made aware of participant responsibilities and legal obligations. Finally, you’ll be provided with support and logistical information you’ll need to join the community.
+  - **What will I learn during onboarding process?** There will be an overview of the CNTT mission and goals, community organizational structures and processes, as well as an introduction to the technology development status and roadmaps. Additionally, participant responsibilities and legal obligations with be covered. Finally, you will be introduced to the support and logistical information needed to participate fully in the CNTT community.
```

Comment:
typo: "... during onboarding process?" --> "... during the onboarding process?"

Reply:
done